Helm/opensearch-dashboards/templates/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "opensearch-dashboards.fullname" . }}-config
+  labels: {{ include "opensearch-dashboards.labels" . | nindent 4 }}
+data:
+{{- range $configName, $configYaml := .Values.config }}
+  {{ $configName }}: |
+    {{- toYaml $configYaml | nindent 4 }}
+{{- end -}}
```

Comment:
I think this trailing `-}}` should be just `}}` to avoid breaking future additions to the configmap.

Reply:
Changes made in the PR